README.md

```diff
@@ -1,6 +1,8 @@
 # plotsbot
 [![Build Status](https://travis-ci.org/publiclab/plotsbot.svg?branch=master)](https://travis-ci.org/publiclab/plotsbot)
 [![license](https://img.shields.io/badge/license-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com/)
+
```

Comment:
You accidentally added an extra newline and it looks kind of excessive. If it does not trouble you, could you remove it? This PR looks awesome otherwise.